### PR TITLE
UI branding

### PR DESCRIPTION
**Pull request by carlito-scality**
-https://github.com/scality/metalk8s/issues/775

To test we can override the public/brand folder by respecting the naming of files. All these files are required.

 you will need https://github.com/scality/metalk8s/issues/820 if you want to override brand from a "volume"

### CODE REVIEW

File: ui/src/services/api.js [hunk 1]
```diff
@@ -34,6 +34,14 @@ export async function getNodes() {
   try {
     return await coreV1.listNode();
   } catch (error) {
-    console.error('Error retrieving nodes', error.body ? error.body : error);
+    return { error };
```
Comment (gdemonet) [inline ui/src/services/api.js:37]: :+1:

**Review by MonPote — DISMISSED** [review]

**Comment by carlito-scality** [comment]
@bert-e approve

**Comment by carlito-scality** [comment]
@bert-e reset

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (NicolasT) [inline yarn.lock]: So, this one is coming back now? :confused:

Comment (carlito-scality) [reply yarn.lock]: weird. => removed

**Review by jbertran — APPROVED** [review]

**Pull request merged**